### PR TITLE
feat: New design for authentication pages

### LINKED ISSUE #4227: feat: Design update for Authentication pages

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

Fixes #4227 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

**Screen recording**

https://user-images.githubusercontent.com/64252451/190315527-a23f653e-3eeb-4748-979a-bf632a98fe5a.mov

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/javascript/dashboard/assets/scss/widgets/_login.scss [hunk 1]
```diff
@@ -1,66 +1,35 @@
...
 
-      .error {
-        font-size: $font-size-small;
-      }
-    }
+.auth-screen--footer {
+  font-size: var(--font-size-small);
+  font-weight: var(--font-weight-medium);
+  margin-bottom: var(--space-two);
+  margin-top: var(--space-three);
```
Comment (iamsivin) [inline app/javascript/dashboard/assets/scss/widgets/_login.scss:22]:
```suggestion
  margin: var(--space-two) 0;
```

File: app/javascript/dashboard/i18n/locale/en/signup.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "REGISTER": {
-    "TRY_WOOT": "Register an account",
-    "TITLE": "Register",
+    "TITLE": "Sign Up",
+    "TRY_WOOT": "Create your chatwoot account",
```
Comment (iamsivin) [inline app/javascript/dashboard/i18n/locale/en/signup.json:4]:
```suggestion
    "TRY_WOOT": "Create your Chatwoot account",
```

File: app/javascript/dashboard/i18n/locale/en/login.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
   "LOGIN": {
     "TITLE": "Login to Chatwoot",
+    "DESCRIPTION": "login to your chatwoot account",
```
Comment (iamsivin) [inline app/javascript/dashboard/i18n/locale/en/login.json:4]:
```suggestion
    "DESCRIPTION": "Login to your Chatwoot account",
```

File: app/javascript/dashboard/assets/scss/widgets/_login.scss [hunk 4]
```diff
@@ -1,66 +1,34 @@
-.auth-wrap {
-  width: 100%;
+.auth-screen--view {
+  display: flex;
+  justify-content: center;
+  height: 100%;
 }
```
Comment (iamsivin) [inline app/javascript/dashboard/assets/scss/widgets/_login.scss:5]:
```suggestion
.auth-screen--view {
  display: flex;
  height: 100%;
  justify-content: center;
}
```

File: app/javascript/dashboard/routes/auth/components/AuthInput.vue [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+      type: String,
+      default: '',
+    },
+    error: {
+      type: String,
+      default: '',
+    },
+    readonly: {
+      type: Boolean,
+      deafaut: false,
```
Comment (pranavrajs) [inline app/javascript/dashboard/routes/auth/components/AuthInput.vue:58]:
```suggestion
      default: false,
```

**Comment by pranavrajs** [comment]
@iamsivin Please remove the SVGs here and use the actual DOM element to show the title, subtitle, and testimonial content.

**Comment by pranavrajs** [comment]
I am closing this since we have the signup page merged. We will pick the rest of the changes as smaller enhancements later.

**Pull request closed without merging**